### ISSUE TITLE
calculate distillation loss

### ISSUE THREAD

**Issue by as754770178** [Tencent/PocketFlow#54]
The `calc_loss` function in `distillation_helper.py` that take primary model's logits(logits of the full-precision uncompressed network) as logits and distilled model's logits as labels, is it right? I think should take primary model's logits as labels and distilled model's logits as logits

**Comment by jiaxiang-wu** [comment]
You may have misunderstood the meaning of "primary model" and "distilled model".
In the `calc_loss` function of `DistillationHelper` class, "primary model" refers to the student model (or compressed model) and "distilled models" refers to the teacher model (or full-precision model), since its outputs/activations/knowledge are/is distilled to generate extra supervision for the student network.

**Comment by as754770178** [comment]
OK, thanks
